packages/monitor-v2/src/bot-oo-v3/common.ts

```diff
@@ -0,0 +1,115 @@
+import { getMnemonicSigner, getRetryProvider } from "@uma/common";
+import { Signer } from "ethers";
+
+import type { Provider } from "@ethersproject/abstract-provider";
+
+export { OptimisticOracleV3Ethers } from "@uma/contracts-node";
+export { Logger } from "@uma/financial-templates-lib";
+export { getContractInstanceWithProvider } from "../utils/contracts";
+
+export interface BotModes {
+  settleAssertionsEnabled: boolean;
+}
+
+export interface BlockRange {
+  start: number;
+  end: number;
+}
+
+export interface MonitoringParams {
+  provider: Provider;
+  chainId: number;
+  runFrequency: number;
+  botModes: BotModes;
+  signer: Signer;
+  warmingUpBlockLookback: number;
+  blockLookback: number;
+  maxBlockLookBack: number;
+  firstRun?: boolean;
+}
+
+const blockDefaults = {
+  "1": {
+    // Mainnet
+    day: 576,
+    month: 17280,
+    maxBlockLookBack: 7000,
+  },
+  "137": {
+    // Polygon
+    day: 4320,
+    month: 129600,
```

Comment:
Is this supposed to be the number of blocks in a month?

I thought this value was ~1314000 (10x) what you have here, since polygon is about 1 block per 2 seconds.